src/App.jsx

```diff
@@ -1,13 +1,214 @@
+    });
+
+    // handleClick zorgt voor het op- en aftellen van de counters
+    // waarde -1 of +1 wordt meegegeven afhankelijk van de button
+    function handleClick(counterName, newValue) {
+        setCounterState({
+            ...counterState,
+            [counterName]: (counterState[counterName] + newValue)
+        })
+    }
+
+    function resetCounters() {
+        // Dynamisch resetten van de counters - ongeacht het aantal in het state-object,
+        // Object.keys leest het object uit in een array om overheen te mappen
+        // Object.fromEntries maakt van de array weer een object zoals de state terugverwacht
+        setCounterState((prevState) =>
+            Object.fromEntries(Object.keys(prevState).map((key) => [key, 0]))
+        );
+        // Handmatig resetten van de counters
+        // setCounterState({
+        //     bananaCounter: 0,
+        //     strawberryCounter: 0,
+        //     appleCounter: 0,
+        //     kiwiCounter: 0
+        // });
+    }
+
+    function handleChange(e) {
+        const changedFieldName = e.target.name;
+        console.log(e.target.value);
+        const newValue = e.target.type === "checkbox" ? e.target.checked : e.target.value;
+
+        setFormState({
+            ...formState,
+            [changedFieldName]: newValue,
+        });
+    }
+
+    // Verzend button logt zowel formState als counterState in de terminal
+    function handleSubmit() {
+        console.log(formState);
+        console.log(counterState);
+    }
+
+    return (
+        <>
+            <h1>Fruitmand bezorgservice</h1>
+
+            <section>
+                <FruitCounter
+                    icon='🍓'
+                    title='Aardbeien'
+                    fruitType='strawberry'
+                    counter={counterState.strawberryCounter}
+                    handleClick={handleClick}
+                />
+                <FruitCounter
+                    icon='🍌'
+                    title='Bananen'
+                    fruitType='banana'
+                    counter={counterState.bananaCounter}
+                    handleClick={handleClick}
+                />
+                <FruitCounter
+                    icon='🍏'
+                    title='Appels'
+                    fruitType='apple'
+                    counter={counterState.appleCounter}
+                    handleClick={handleClick}
+                />
+                <FruitCounter
+                    icon='🥝'
+                    title="Kiwi's"
+                    fruitType='kiwi'
+                    counter={counterState.kiwiCounter}
+                    handleClick={handleClick}
+                />
+
+                <Button
+                    type='button'
+                    title='Reset'
+                    onclick={resetCounters}
+                />
+            </section>
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+            <form className="order-form">
+                <p>
```

Comment:
Semantisch gezien is het gebruik van een p element om elke input niet ideaal. Het p element is namelijk bedoelt voor paragrafen of tekstuele inhoud. Je had hier beter een div of een span voor kunnen gebruiken